### ISSUE TITLE
No classes for task list elements with remark-gfm plugin

### ISSUE THREAD

**Issue by MartialSeron** [remarkjs/react-markdown#522]
### Subject of the issue

No classes for task list elements with [remark-gfm](https://github.com/remarkjs/remark-gfm) plugin like [specified in the doc](https://github.com/remarkjs/remark-gfm#use)

### Your environment

*   **Packages**: react-markdown, remark-gfm

### Steps to reproduce

Codesandbox : https://codesandbox.io/s/react-markdown-x3bzd?file=/src/App.js

markdown text is 

```
* [ ] todo
* [x] done
```

### Expected behavior

```html
<ul class="contains-task-list">
  <li class="task-list-item">
    <input type="checkbox" readonly>todo
  </li>
  <li class="task-list-item">
    <input type="checkbox" readonly checked>done
  </li>
</ul>
```

### Actual behavior

The classes are missing for `<ul>` and `<li>`

```html
<ul>
  <li>
    <input type="checkbox" readonly>todo
  </li>
  <li>
    <input type="checkbox" readonly checked>done
  </li>
</ul>
```

This way we can't add specific style (eg : list-style-type: none; ) for task list

**Comment by wooorm** [comment]
Those classes are not added by that plugin, but by `remark2rehype` in that example.

In the next major, we plan on using that plugin too.

For now you can overwrite the list item renderer, using something [like this](https://github.com/remarkjs/react-markdown/blob/634f88d610c3beb39411db5c928d2cfff440f9ff/src/renderers.js#L82-L90) but adding a class.

**Comment by wooorm** [comment]
This should be solved by landing GH-563 today, which will soon be released in v6.0.0!
